packages/open-truss/src/configuration/engine-v1/Frame.tsx

```diff
@@ -163,3 +254,22 @@ export function getComponent(
 
   return Component
 }
+
+export function getComponentProps(
```

Comment:
I feel like we could combine `getComponent` and `getComponentProps`. One idea is to have `getComponent` return the component and `Props` if it exists and then we could pass `Props` into `processProps` up above.